### PR TITLE
Switch macos to popos

### PR DESCRIPTION
**Pull request by storyaddict**
Changes have been made. Ready to merge after final review.

### CODE REVIEW

File: _articles/Pop!_OS-20.04-LTS-Release-Notes.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Pop!\_Shop now loads up to 20 Pop!\_Picks making highlighted applications more a
 Dark mode is enabled by default. Default desktop wallpaper has a new design. Both defaults can be changed within the Appearance settings.
 
 ### GNOME 3.36
+<<<<<<< HEAD:_articles/Pop!_OS-20.04-LTS-Release-Notes.md
```
Comment (jackpot51) [inline _articles/Pop!_OS-20.04-LTS-Release-Notes.md:60]: Why are there merge conflicts committed?

Comment (storyaddict) [reply _articles/Pop!_OS-20.04-LTS-Release-Notes.md:60]: I did not see/wasn't alerted to the conflict. I didn't make any edits to the release notes article.

Comment (storyaddict) [reply _articles/Pop!_OS-20.04-LTS-Release-Notes.md:60]: Or, I didn't see them until now I should say.

Comment (jackpot51) [reply _articles/Pop!_OS-20.04-LTS-Release-Notes.md]: It's fixed now, thanks

**Review by Socialhappiness — APPROVED** [review]
Thanks for taking the suggestions into account. This article looks great. The one thing I would add at the end is a little call to action to invite them to install Pop with a link... if you want!

**Review by ahoneybun — APPROVED** [review]
Looks amazing to me!

**Pull request merged**